cwf/gateway/docker/c/Dockerfile

```diff
@@ -82,78 +65,66 @@ RUN apt-get -y update && apt-get -y install \
+RUN bazel build //lte/gateway/c/session_manager:sessiond \
+  # Sentry isn't supported on this base image (breakpad will not compile)
+  --define=disable_sentry_native=1 \
+  --define=folly_so=1 \
+  --disk_cache=/tmp/bazel-cache \ 
+  --repository_cache=/tmp/bazel-cache-repo \
+  --verbose_failures
 
 # -----------------------------------------------------------------------------
 # Dev/Production image
 # -----------------------------------------------------------------------------
 FROM $OS_DIST:$OS_RELEASE AS gateway_c
-ARG OS_DIST
-ARG OS_RELEASE
-ARG EXTRA_REPO
 
-# Add the magma apt repo
-RUN apt-get update && \
-    apt-get install -y apt-utils software-properties-common apt-transport-https gnupg
-COPY orc8r/tools/ansible/roles/pkgrepo/files/jfrog.pub /tmp/jfrog.pub
-
-RUN apt-key add /tmp/jfrog.pub && \
-    apt-add-repository "deb $EXTRA_REPO $OS_RELEASE main"
-
-
-# Install runtime dependencies
-RUN apt-get -y update && apt-get -y install \
-  curl \
-  sudo \
-  # install prometheus
-  prometheus-cpp-dev \
-  # install openvswitch
-  magma-libfluid \
-  # install lxml
-  python3-lxml \
-  bridge-utils \
-  # install yaml parser
-  libyaml-cpp-dev \
-  libgoogle-glog-dev \
-  # folly deps
-  libfolly-dev \
-  libdouble-conversion-dev \
-  libboost-chrono-dev \
-  libboost-context-dev \
-  libboost-program-options-dev \
-  libboost-filesystem-dev \
-  libboost-regex-dev \
-  nlohmann-json-dev \
-  redis-server \
-  python-redis \
-  magma-cpp-redis \
-  grpc-dev \
-  protobuf-compiler \
-  libprotoc-dev \
-  netcat
+# Copy runtime dependencies
+COPY --from=builder /usr/lib/x86_64-linux-gnu/libboost* /usr/lib/x86_64-linux-gnu/
+COPY --from=builder /usr/lib/x86_64-linux-gnu/libevent-* /usr/lib/x86_64-linux-gnu/
+COPY --from=builder /usr/lib/x86_64-linux-gnu/libgflags* /usr/lib/x86_64-linux-gnu/
+COPY --from=builder /usr/lib/x86_64-linux-gnu/libglog* /usr/lib/x86_64-linux-gnu/
+COPY --from=builder /usr/lib/x86_64-linux-gnu/libsnappy.* /usr/lib/x86_64-linux-gnu/
+COPY --from=builder /usr/lib/x86_64-linux-gnu/libdouble-conversion.* /usr/lib/x86_64-linux-gnu/
+COPY --from=builder /usr/lib/x86_64-linux-gnu/libicui18n.* /usr/lib/x86_64-linux-gnu/
+COPY --from=builder /usr/lib/x86_64-linux-gnu/libicuuc.* /usr/lib/x86_64-linux-gnu/
+COPY --from=builder /usr/lib/x86_64-linux-gnu/libicudata.* /usr/lib/x86_64-linux-gnu/
+COPY --from=builder /usr/lib/x86_64-linux-gnu/libssl* /usr/lib/x86_64-linux-gnu/
+COPY --from=builder /usr/lib/x86_64-linux-gnu/openssl* /usr/lib/x86_64-linux-gnu/
+COPY --from=builder /usr/lib/x86_64-linux-gnu/libcrypto* /usr/lib/x86_64-linux-gnu/
+COPY --from=builder /usr/lib/x86_64-linux-gnu/libunwind* /usr/lib/x86_64-linux-gnu/
+
+COPY --from=builder /usr/local/lib/libfolly.so /usr/local/lib/
+
+RUN ldconfig 2> /dev/null
 
 # Copy the build artifacts.
-COPY --from=builder /build/c/session_manager/sessiond /usr/local/bin/sessiond
+COPY --from=builder /magma/bazel-bin/lte/gateway/c/session_manager/sessiond /usr/local/bin/sessiond
```

Comment:
Couldn't spot an apt-get clean (or similar) command at the end. It helps clean up any packages that are left in the local cache and are not needed anymore. It probably wouldn't help with build times, but could help in reducing image size.